addons/account/models/account_move_line.py

```diff
@@ -415,6 +415,12 @@ class AccountMoveLine(models.Model):
              "associated partner",
     )
     is_refund = fields.Boolean(compute='_compute_is_refund')
+    # === Check Info === #
+    check_number = fields.Char(
```

Comment:
Don't we also want to be able to search on it easily?
Or is it already doable?

Reply:
I don't get what you referring too. Sorry could you elaborate?

Reply:
In other words, don't we needs a `search` method defined on the field?
https://www.odoo.com/documentation/17.0/developer/reference/backend/orm.html?highlight=orm#computed-fields